src/js/models/v2/awards/financialSystemDetails/BaseFinancialSystemDetailsRow.js

```diff
@@ -12,8 +12,10 @@ export const formatDate = (date) => date.format('MM/DD/YYYY');
 const BaseFinancialSystemDetailsRow = {
     populate(data) {
         this.id = data.financial_accounts_by_awards_id || '';
+        this.reporting_fiscal_year = data.submission.reporting_fiscal_year || null;
+        this.reporting_fiscal_quarter = data.submission.reporting_fiscal_quarter || null;
         this._submissionDate = (
-            (data.certified_date && parseDate(data.certified_date)) || null
+            (data.submission.reporting_fiscal_year && data.submission.reporting_fiscal_quarter) || null
```

Comment:
`this._submissionDate` is an unnecessary variable because you can check for the existence of `this._year` and `this._quarter` in the `submissionDate` getter function.